src/main/java/org/jitsi/jicofo/FocusManager.java

```diff
@@ -289,7 +300,9 @@ public void start()
             = new JitsiMeetServices(
                     protocolProviderHandler,
                     jvbProtocolProvider,
-                    focusUserDomain);
+                    focusUserDomain,
+                    JidCreate.domainBareFrom(
+                        conferenceMucPrefix + "." + xmppDomainConfig));
```

Comment:
If we're matching the complete domain, why not get rid of the discovery logic? Just pass the configuration to `Health`.

Reply:
+1

Reply:
Ok, make sense :)

Reply:
How about now? 
Now this PR removes more than it adds :)